api/notd/api/api_v1.py

```diff
@@ -81,6 +82,11 @@ async def get_collection_by_address(registryAddress: str):  # request: RetreiveC
         collection = await notdManager.get_collection_by_address(address=registryAddress)
         return GetCollectionResponse(collection=(await responseBuilder.collection_from_model(collection=collection)))
 
+    @router.get('/collections/{registryAddress}/holdings', response_model=ListHoldingsForCollectionResponse)
+    async def list_collection_holdings(registryAddress: str, ownerAddress: str):  # request: ListHoldingsForCollectionRequest
```

Comment:
how does ownerAddress get populated here?

Reply:
curl -X GET -H "Accept: application/json" "http://localhost:5000/v1/collections/0xfE8C6d19365453D26af321D0e8c910428c23873F/holdings?ownerAddress=0x18090cDA49B21dEAffC21b4F886aed3eB787d032"

Reply:
actually sorry can we change the url to `/collections/{..}/tokens/owner/{ownerAddress}` and call the function list_collection_tokens_by_owner

Reply:
Oh thought you wanted it as a query, will change it back

Reply:
i did sorry just changed it now cos i reaalised the caching would be wierd